sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -2715,4 +2715,70 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase with ExecutionE
         "numFields" -> numFields.toString,
         "schemaLen" -> schemaLen.toString))
   }
+
+  def couldNotReadOffsetRangeInKafkaError(startOffset: Long,
+                                          endOffset: Long,
```

Comment:
nit indentation : )
`For method declarations, use 4 space indentation ` from https://github.com/databricks/scala-style-guide

Reply:
Yeah, i need to fix it every where. IntelliJ messed it up. Thanks